__tests__/news/news.mutation.test.js

```diff
@@ -60,9 +62,14 @@ const existingNews = {
 };
 
 let newsId = '';
+let token;
 const newsDoesNotExistId = '1f2ad410eb01783384e6111b';
 
 describe('test news mutations', () => {
```

Comment:
Please, create 3 `describe` statements for `Create` tests, `Update` tests, `Delete` tests to increase the readability of the tests

Reply:
Please, also add here tests for permissions, that simple users without admin permissions or even without token cannot add, update or delete materials and check that proper error messages are returned.

Reply:
You're right! Changed!